app/Http/Transformers/PostTransformer.php

```diff
@@ -53,6 +52,11 @@ public function transform(Post $post)
             'updated_at' => $post->updated_at->toIso8601String(),
         ];
 
+        // If the post isn't anonymous or if this is the owner of the post, return PII information.
```

Comment:
Sorry, can we re-word this? I think saying "PII" here is a little misleading (since we already lock down most personally-identifiable information here and in Northstar based on the user's privileges).

> If this post is for an anonymous action (and viewer is not owner/staff), hide the user ID.

Reply:
yes great call!